boltdesignsystem/bolt
# DS-782 Refactor dialog close button

## Jira

https://pegadigitalit.atlassian.net/browse/DS-782

## Summary

Refactored dialog's close button and added the `transparent` prop.

## Details

1. The close button is positioned using CSS grid rather than absolute position.
1. `transparent` prop added to enhance image/video only dialogs.
1. Revised all dialog docs.

## How to test

Run the branch locally and go through all the dialog docs. Proofread everything and test to see if all props work as intended.
